Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -114,6 +126,7 @@ func NewCmdGet(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string, options *GetOptions) error {
 	selector := cmdutil.GetFlagString(cmd, "selector")
 	allNamespaces := cmdutil.GetFlagBool(cmd, "all-namespaces")
+	allKinds := cmdutil.GetFlagBool(cmd, "show-kind")
```

Comment:
I think the naming here is confusing. Why is this called `allKinds` while the flag is `show-kind`?
Then later we say `printWithKinds := allKinds`. That's confusing to follow, at least for me.

It seems that we don't need both `allKinds` and `printWithKinds`.

How about:

```go
showKind := cmdutil.GetFlagBool(cmd, "show-kind")
// ...

if mustPrintWithKinds(objs, infos, sorter) {
	showKind = true
}
```

The variable name follows the flag name, and we force-set it to `true` if we must.